cli/main.rs

```diff
@@ -112,14 +112,28 @@ fn main() {
     // Setup runtime.
     js_check(main_worker.execute("denoMain()"));
 
-    // Execute main module.
-    if let Some(main_module) = state.main_module() {
-      debug!("main_module {}", main_module);
-      js_check(main_worker.execute_mod(&main_module, should_prefetch));
-      if should_display_info {
-        // Display file info and exit. Do not run file
-        main_worker.print_file_info(&main_module);
-        std::process::exit(0);
+    if state.flags.eval {
+      // Wrap provided script in async function so asynchronous methods
+      // work. This is required until top-level await is not supported.
+      let js_source = format!(
+        "async function __eval(){{
+          {}
+        }}
+        __eval();
+        ",
+        &state.argv[1]
+      );
+      js_check(main_worker.execute(&js_source));
```

Comment:
execute implies that imports are not allowed. This might be surprising. That said, I don't think there's a way to "execute_mod" without loading a file right now. Add a TODO here 

// TODO Support ES modules once Worker supports evaluating anonymous modules.